src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -607,78 +604,84 @@ private void PopulateSourceRepoList(string optionalSelectSourceName = null)
         /// <summary>
         /// This method is called from several event handlers. So, consolidating the use of JTF.Run in this method
         /// </summary>
-        internal void SearchPackagesAndRefreshUpdateCount(string searchText, bool useCacheForUpdates)
+        internal void SearchPackagesAndRefreshUpdateCount(bool useCacheForUpdates)
```

Comment:
if you are making this function use a private variable, then it would be better if you make the function private too.

Reply:
is there a standard norm around this? Internal also seemed fine and allows us to write Apex test tomorrow